Annotation/Service.php

```diff
@@ -53,4 +53,10 @@
 
     /** @var array<string> */
     public $environments = array();
+
+    /** @var boolean */
+    public $autowire;
+
+    /** @var array<string> */
+    public $autowiring_types = array();
```

Comment:
I'd say camel case and we should deprecate any attributes snack cased. 
If I'm not wrong all majors libraries using annotations use camel case right?